<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodePropertyDlg</class>
 <widget class="QDialog" name="NodePropertyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Passed Station</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
          <item row="6" column="1">
           <widget class="QIndexComboBox" name="comboImgListPool_P"/>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spPosX_P">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Pos X</string>
            </property>
            <property name="buddy">
             <cstring>spPosX_P</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelAngle_P">
            <property name="text">
             <string>Angle</string>
            </property>
            <property name="buddy">
             <cstring>spAngle_P</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Pos Y</string>
            </property>
            <property name="buddy">
             <cstring>spPosY_P</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spPosY_P">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboPivot_P"/>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboType_P"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pivot Point</string>
            </property>
            <property name="buddy">
             <cstring>comboPivot_P</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Image Refer to</string>
            </property>
            <property name="buddy">
             <cstring>comboType_P</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spAngle_P">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Image List Item</string>
            </property>
            <property name="buddy">
             <cstring>comboImgListPool_P</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QIndexComboBox" name="comboDisplayPool_P"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Display Pool Item</string>
            </property>
            <property name="buddy">
             <cstring>comboDisplayPool_P</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QCheckBox" name="chkUseDefaultSpot_P">
            <property name="text">
             <string>Using Default Node Icon</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Target Station</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Pos X</string>
            </property>
            <property name="buddy">
             <cstring>spPosX_T</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spPosX_T">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spPosY_T">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelAngle_T">
            <property name="text">
             <string>Angle</string>
            </property>
            <property name="buddy">
             <cstring>spAngle_T</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Pos Y</string>
            </property>
            <property name="buddy">
             <cstring>spPosY_T</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboType_T"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Pivot Point</string>
            </property>
            <property name="buddy">
             <cstring>comboPivot_T</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboPivot_T"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Image Refer to</string>
            </property>
            <property name="buddy">
             <cstring>comboType_T</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spAngle_T">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Image List Item</string>
            </property>
            <property name="buddy">
             <cstring>comboImgListPool_T</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QIndexComboBox" name="comboImgListPool_T"/>
          </item>
          <item row="5" column="1">
           <widget class="QIndexComboBox" name="comboDisplayPool_T"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Display Pool Item</string>
            </property>
            <property name="buddy">
             <cstring>comboDisplayPool_T</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QCheckBox" name="chkUseDefaultSpot_T">
            <property name="text">
             <string>Using Default Node Icon</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Remain Station</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,1">
          <item row="6" column="1">
           <widget class="QIndexComboBox" name="comboImgListPool_R"/>
          </item>
          <item row="5" column="1">
           <widget class="QIndexComboBox" name="comboDisplayPool_R"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Image List Item</string>
            </property>
            <property name="buddy">
             <cstring>comboImgListPool_R</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spPosX_R">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Pos Y</string>
            </property>
            <property name="buddy">
             <cstring>spPosY_R</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Pos X</string>
            </property>
            <property name="buddy">
             <cstring>spPosX_R</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spPosY_R">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Pivot Point</string>
            </property>
            <property name="buddy">
             <cstring>comboPivot_R</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spAngle_R">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelAngle_R">
            <property name="text">
             <string>Angle</string>
            </property>
            <property name="buddy">
             <cstring>spAngle_R</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboPivot_R"/>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboType_R"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Image Refer to</string>
            </property>
            <property name="buddy">
             <cstring>comboType_R</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Display Pool Item</string>
            </property>
            <property name="buddy">
             <cstring>comboDisplayPool_R</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QCheckBox" name="chkUseDefaultSpot_R">
            <property name="text">
             <string>Using Default Node Icon</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="0,1,0,3">
     <item>
      <widget class="QLabel" name="labelStationInfo">
       <property name="text">
        <string>Related Station</string>
       </property>
       <property name="buddy">
        <cstring>comboStationInfo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QIndexComboBox" name="comboStationInfo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QPushButton" name="btnCopyAll">
         <property name="text">
          <string>Copy All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QIndexComboBox</class>
   <extends>QComboBox</extends>
   <header>qindexcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spPosX_P</tabstop>
  <tabstop>spPosY_P</tabstop>
  <tabstop>spAngle_P</tabstop>
  <tabstop>comboPivot_P</tabstop>
  <tabstop>comboType_P</tabstop>
  <tabstop>comboDisplayPool_P</tabstop>
  <tabstop>comboImgListPool_P</tabstop>
  <tabstop>chkUseDefaultSpot_P</tabstop>
  <tabstop>spPosX_T</tabstop>
  <tabstop>spPosY_T</tabstop>
  <tabstop>spAngle_T</tabstop>
  <tabstop>comboPivot_T</tabstop>
  <tabstop>comboType_T</tabstop>
  <tabstop>comboDisplayPool_T</tabstop>
  <tabstop>comboImgListPool_T</tabstop>
  <tabstop>chkUseDefaultSpot_T</tabstop>
  <tabstop>spPosX_R</tabstop>
  <tabstop>spPosY_R</tabstop>
  <tabstop>spAngle_R</tabstop>
  <tabstop>comboPivot_R</tabstop>
  <tabstop>comboType_R</tabstop>
  <tabstop>comboDisplayPool_R</tabstop>
  <tabstop>comboImgListPool_R</tabstop>
  <tabstop>chkUseDefaultSpot_R</tabstop>
  <tabstop>comboStationInfo</tabstop>
  <tabstop>btnCopyAll</tabstop>
  <tabstop>btnApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
